app/controllers/alarms_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
   def create
     @alarm = current_user.alarms.create(alarm_params)
     if @alarm.valid?
-      hours, minutes = params[:alarm][:time].split(':')
+      hours, minutes = alarm_params[:time].split(':')
```

Comment:
I highly recommend assigning just one value per line.

Reply:
Ok, I check that, thanks